civihr_employee_portal/src/Helpers/ContactImagePathfinder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\Helpers;
+
+class ContactImagePathfinder {
+  /**
+   * @param $contactID
+   *   The ID of the target contact
+   *
+   * @return string|NULL
+   *   The image path if it exists, or NULL if not set or doesn't exist.
+   */
+  public static function getPathToContactImage($contactID) {
```

Comment:
there are some repeated words between the class and this method names, I would either change the class name or the method name : 

Example 1)  ImagePathfinder::getPathToContactImage()

Example 2) ContactImagePathfinder:getPath();

what do you think ?

Reply:
I think you're right. It's a very specific class, but that's not necessarily a bad thing. I prefer the second one because it's clear to and dev looking at php files what services are available